project.clj

```diff
@@ -89,4 +89,8 @@
                               [:name "Ryan Brush"]
                               [:url "http://www.clara-rules.org"]]]
   :deploy-repositories [["snapshots" {:url "https://oss.sonatype.org/content/repositories/snapshots/"
-                                      :creds :gpg}]])
+                                      :creds :gpg}]
+                        ["clojars" {:url "https://repo.clojars.org"
+                                    :username :env/clojars_username
+                                    :password :env/clojars_password
+                                    :sign-releases false}]])
```

Comment:
I suppose this isn't completely ideal, but since we (at least I) haven't published the GPG key I used to manually deploy in reality it isn't much different from what we've done before.  Also in practice it would be quite obvious if releases that didn't have a corresponding tag were created and since nobody has ever asked about signature on the jars I'm guessing it isn't checked.  So basically I think this is OK.

Reply:
I thought this would be fine as well, if someone has an issue with jar signing in the future we can open a new issue and find a way to get GPG working on the CI environment but I don't think that will be the case.